### DISCUSSION TITLE
how to List  content item(s) owned by login user.

### DISCUSSION THREAD

**Discussion by jnucpz** [OrchardCMS/OrchardCore#13577] [Q&A]
In release 1.4, On the Manage Content screen, it can list my own content items though setting the permission ( List content item(s) owned by all users )。but release 1.6, it list all contentitems, include other content item. i want to only list my own content items. how can i work it? the permission remove （List content item(s) owned by all users ）。

**Comment by MikeAlhayek** [comment]
you should be able to use `status:Owner` to only see content that are owned by current user

**Reply by jnucpz** [reply]
i don't want to use filter , any other resolution ?

**Reply by MikeAlhayek** [reply]
If the logged user has a role 'X', you can give role X permission to view their own content items only.

**Reply by jnucpz** [reply]
<img width="1025" alt="微信图片_20230421115813" src="https://user-images.githubusercontent.com/17457950/233537881-4e8d5e4c-6165-41a6-ab93-32212a9e8a70.png">
 The default permission is 
View all content . i can't stop it.

**Reply by MikeAlhayek** [reply]
Sure you can. Edit both the Authenticated and Anonymous role and remove "View content" permission. If you have a specific content type that you want to expose publicly like SitePage or something like that, you may make it securable and then grant View SitePage content to Anonymous role. 

Not edit your X role, and grant the View own content + list content permissions you want to see. That should give you the behavior you are looking for

**Reply by jnucpz** [reply]
thanks, I download the latest release. it works. it can remove view all content permission.
